src/identitylookup.cs

```diff
@@ -15,6 +16,8 @@
 namespace IdentityLookup {
 
 	[iOS (11,0)]
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
```

Comment:
Do we use [Introduced] for external APIs and [platform (version,version)] for internal (?) APIs?